### PR TITLE
Authorization Middleware 

### PR DESCRIPTION
**Pull request by Fatmasiam**
#88

### CODE REVIEW

**Review by Angham116 — APPROVED** [review]

**Review by Anies12 — APPROVED** [review]

**Review by ahmedalmi95 — APPROVED** [review]

File: src/controllers/index.js [hunk 1]
```diff
@@ -4,10 +4,12 @@ const signup = require('./signup');
 const specialize = require('./specialization');
 const login = require('./login');
 const error = require('./error');
+const authorizatin = require('../middlewares/authorization');
 
 const router = express.Router();
 
 router.get('/login', login.get);
+router.use(authorizatin);
```
Comment (kannanhassouna) [inline src/controllers/index.js]: typo
```suggestion
router.use(authorization );
```

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

File: src/controllers/index.js (hunk 1 above)
Comment (kannanhassouna) [reply src/controllers/index.js]: edit both the module name (alias) and name of the file

Comment (Fatmasiam) [reply src/controllers/index.js]: @knanahassouna1  done for the alias name

File: src/controllers/index.js [hunk 2]
```diff
@@ -4,10 +4,12 @@ const signup = require('./signup');
...
 const login = require('./login');
 const error = require('./error');
+const authorization = require('../middlewares/authorization');
 
 const router = express.Router();
 
 router.get('/login', login.get);
+router.use(authorization);
 router.post('/specialize', specialize);
 router.route('/signup')
```
Comment (kannanhassouna) [inline src/controllers/index.js:13]: re-oreder you routes and put login and sign up before ``` auth ``` middleware

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

File: src/controllers/index.js (hunk 2 above)
Comment (Fatmasiam) [reply src/controllers/index.js:13]: @knanahassouna1  done

**Review by kannanhassouna — APPROVED** [review]

**Pull request merged**